src/test/bin/quic_gtest.cpp

```diff
@@ -1769,7 +1770,8 @@ TEST_P(WithSendArgs1, Send) {
             GetParam().UseSendBuffer,
             GetParam().UnidirectionalStreams,
             GetParam().ServerInitiatedStreams,
-            false); // FifoScheduling
+            false,  // FifoScheduling
+            true); // SendUdpToQtipListener
```

Comment:
This test will have a different behavior in UM and KM, are we OK with that? Or should we create a new test for this that runs only in UM since it doesn't make sense in KM?

Reply:
So the behavior for the test will be the same even if set `SendUdpToQtipListener` to `1` in KM because 
![image](https://github.com/user-attachments/assets/8a4029b4-c0a9-4888-90de-d4edf8c074d3)

Reply:
Ok. If the result is the same, I'd consider keeping the parameters consistent just to avoid potential weird bugs in the future (let's say we decide to support QTIP in kernel). It's way to easy to miss a difference between the parameters :)

But at this point, I nitpick, either is ok.